### ISSUE TITLE
Graph module: collection.insert() not honoring graph definitions

### ISSUE THREAD

**Issue by sinfex** [arangodb/arangodb#1831]
According to documentation for 2.8.6 on [Collection Methods](https://docs.arangodb.com/Documents/DocumentMethods.html):
> collection.insert is an alias for collection.save

When using the Graph module, however, only `collection.save()` actually honors graph definitions; `collection.insert()` does not seem to be wrapped.

```
const graphs = require('@arangodb/general-graph');
const graph = graphs._create('testGraph');
graph._extendEdgeDefinitions(graphs._relation('userTasks', 'users', 'tasks'));
graph.userTasks.save('tasks/1', 'tasks/1', {}); // throws exception
graph.userTasks.insert('tasks/1', 'tasks/1', {}); // doesn't throw exception
```

**Comment by mchacki** [comment]
Thank you very much for spotting this. Is now fixed in devel and will be released with 3.0.1

**Comment by mchacki** [comment]
And it will also be included in 2.8.
